horace_core/sqw/@sqw/noisify.m

```diff
@@ -14,13 +14,17 @@
 %           Add noise with Poisson distribution, where the mean value at
 %           a point is the value of pixel signal.
 %
+% Modified to use the object paging functionality. The "noisify" overload
```

Comment:
Well, it might matter what it used to do. People revisit code and it is not immediately obvious to them that life has changed. A bit of history is sometimes helpful in giving context.

Reply:
I strongly disagree, particularly in this case:
- the paging functionality does not change the operation of this function for callers
- for single page operation the internal behaviour is the same
- if a user is interested in previous implementations of a function the Git history has it.

Reply:
Not sure I agree with these. *Planned* operation may not change, but under the axiom that code always contains bugs, actual operation may. "Why has this thing that I have always done now do something different" is a perennial user complaint. On that basis, the internal behaviour for single page operation is also different; it goes through a different set of function calls and code structures. My understanding was that the average user would download from the Wiki or its successor the deployed version (alternatively in IDaaS just start the loaded version) and would not necessarily know how to access GitHub.

Reply:
Having said that I have done my best to represent the available information in a more self-contained form.